docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -100,7 +100,7 @@ A viable configuration file would be:
 
 [source,properties]
 --
-quarkus.datasource.url=jdbc:h2:tcp://localhost/mem:default
+quarkus.datasource.url=jdbc:h2:mem:default
```

Comment:
I don't think that's incorrect. We use a similar one in our integration tests.

Did you have any issue with it?

(Note: we don't support really support embedded H2 as it doesn't work in native mode)